src/app/books.ts

```diff
@@ -2433,12 +2421,208 @@ export const BOOKS : Book[] = [
 			"fimfic", "hardcover", "custom"
 		]
 	}
+	,{
+		"id": 84,
+		"title": "Equestrian Stories",
+		"edition": "1",
+		"img": "https://c1.iggcdn.com/indiegogo-media-prod-cld/image/upload/c_fill,w_695,g_auto,q_auto,dpr_1.0,f_auto,h_460/llnzg3uv6kl7vf9npf4r",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": "2019/04/30",
+		"authors":[
+			{
+				"name": "Nonexistant Publications",
+				"url": "https://www.fimfiction.net/user/129042/Avonder"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/bookshelf/1149322/es1",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://www.indiegogo.com/projects/equestrian-stories-a-short-work-anthology/x/17015185#/",
+				"title": "Hardcover with Dust Cover"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "limited run", "anthology", "That Changeling's a Pony!", "Voyage's End", "The Star In Yellow", "First Contact is Magic", "Dash Tries to Win Your Heart", "Evil is Easy, Governing is Harder", "Raven", "The Incredibly Dense Mind of Rainbow Dash", "Best Hell Ever", "Princess Celestia Gets Mugged", "The Mare at the End of Forever", "Derplicity", "Sunny Skies All Day Long", "Whom the Princesses Would Destroy...", "The Keepers of Discord", "My Little Dashie"
+		]
+	}
+	,{
+		"id": 85,
+		"title": "Off The Mark",
+		"edition": "1",
+		"img": "https://cdn-img.fimfiction.net/story/ckkw-1534612975-415985-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Goldfur",
+				"url": "https://www.fimfiction.net/user/84240/Goldfur"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/415985/off-the-mark",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://www.amazon.com/dp/1797620347",
+				"title": "Perfect-Bound Paperback"
+			}
+		],
+		"tags": [
+			"fimfic", "paperback"
+		]
+	}
+	,{
+		"id": 86,
```

Comment:
Background Pony already exists under id 74. Update that.

Reply:
Done